viz/OSGSegment.cpp

```diff
@@ -290,10 +290,10 @@ osg::ref_ptr<osg::Group> OSGSegment::createVisual(urdf::VisualSharedPtr visual,
                                                                     visual->material->color.g,
                                                                     visual->material->color.b,
                                                                     visual->material->color.a));
-            nodematerial->setSpecular(osg::Material::FRONT,osg::Vec4(0.2,
-                                                                     0.2,
-                                                                     0.2,
-                                                                     1));
+            nodematerial->setSpecular(osg::Material::FRONT,osg::Vec4(visual->material->color.r,
+                                                                    visual->material->color.g,
+                                                                    visual->material->color.b,
+                                                                     0));
```

Comment:
What is the reason for setting alpha of specular to 0?

Reply:
Nothing tbh, I thing it was a leftover testing the overall opacity settings (i can't see any effect changing it), but changes it to use the alphs from the urdf